src/resolvers/teams.js

```diff
@@ -107,8 +107,13 @@ const teamsResolvers = {
       return team;
     },
     async inviteTeam(root, { emails, teamId }) {
-      sendTeamInvites(emails, teamId);
-      return true;
+      try {
+        sendTeamInvites(emails, teamId);
```

Comment:
hmm ok since we're refactoring this, let's do it all the way - add `await` here
```suggestion
        await sendTeamInvites(emails, teamId);
```